### PR TITLE
1311

### PR DESCRIPTION
**Pull request by eparovyshnaya**

### CODE REVIEW

File: bundles/org.eclipse.passage.loc.operator/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -11,4 +11,4 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="0.0.0",
  org.eclipse.passage.loc.workbench;bundle-version="0.0.0",
  org.eclipse.passage.lic.e4.ui;bundle-version="0.0.0"
 Eclipse-BundleShape: dir
-Provide-Capability: licensing.feature;licensing.feature="org.eclipse.passage.loc.operator.product";name="Eclipse Passage Operator";version="2.11.0";provider="Eclispe Passage";level="warn";agreements="licenses/epl-2.0.txt"
+Provide-Capability: licensing.feature;licensing.feature="org.eclipse.passage.loc.operator.product";name="Eclipse Passage Operator";version="2.11.1";provider="Eclispe Passage";level="warn";agreements="licenses/epl-2.0.txt"
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.operator/META-INF/MANIFEST.MF:14]: Please rename corresponding .pub

File: bundles/org.eclipse.passage.lbc.fls.seal/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -15,5 +15,5 @@ Require-Bundle: org.eclipse.passage.lic.api;bundle-version="0.0.0",
  org.eclipse.passage.lic.execute;bundle-version="0.0.0",
  org.eclipse.passage.lic.licenses.model;bundle-version="0.0.0",
  org.eclipse.passage.lic.oshi;bundle-version="0.0.0"
-Provide-Capability: licensing.feature;licensing.feature="org.eclipse.passage.lbc.fls.product";name="Eclipse Passage Floating License Server";version="2.11.0";provider="Eclipse Passage";level="warn"
+Provide-Capability: licensing.feature;licensing.feature="org.eclipse.passage.lbc.fls.product";name="Eclipse Passage Floating License Server";version="2.11.1";provider="Eclipse Passage";level="warn"
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lbc.fls.seal/META-INF/MANIFEST.MF:18]: Please rename corresponding .pub

File: bundles/org.eclipse.passage.loc.operator/META-INF/MANIFEST.MF (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.operator/META-INF/MANIFEST.MF:14]: renamed

File: bundles/org.eclipse.passage.lbc.fls.seal/META-INF/MANIFEST.MF (hunk 2 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lbc.fls.seal/META-INF/MANIFEST.MF:18]: renamed

**Review by ruspl-afed — APPROVED** [review]
LGTM

**Pull request merged**